modules/plot/main/coffee/Axis.coffee

```diff
@@ -573,14 +530,67 @@ class Axis
       xStack
     else Math.max(start, 0)
 
-  # gets the height of the stack for a particular type, group and x value (the seriesId is the series that you want to get the baseline for)
   getYStack: (type, group, x, seriesId, start = 0) ->
+    allSeries = @series()
+    xscaletype = @x.scaleType()
+    yscaledomainmin = @yScale.domainMin
+
     if group
-      yStack = Math.max(@yScale.domainMin, 0)
-      for series, j in @series()
+      yStack = Math.max(yscaledomainmin, 0)
+      maybeys = allSeries.map (series) ->
         if series._.seriesId < seriesId and series.group() == group and series._.type == type
-          ys = series.getY(x, @x.scaleType() is 'discrete')
-          if hx.defined(ys)
-            yStack += ys
-      yStack
-    else Math.max(start, 0)
+          series.getY(x, xscaletype is 'discrete')
+      yStack + hx.sum maybeys.filter hx.identity
+    else
+      Math.max(start, 0)
+
+  calculateYBounds: ->
+    # The series must already be tagged before calling this method
+    # XXX: band series?
```

Comment:
This comment was there before, I'm not sure if it still applies as we explicitly handle BandSeries below